.circleci/docker/Dockerfile

```diff
@@ -0,0 +1,40 @@
+# this Dockerfile is for torchaudio smoke test, it will be created periodically via CI system
+# if you need to do it locally, follow below steps once you have Docker installed
+# assuming you're within the directory where this Dockerfile located
+#  $ docker build . -t torchaudio/smoketest
+
+# if you want to push to aws ecr, make sure you have the rights to write to ECR, then run
+# $ eval $(aws ecr get-login --region us-east-1 --no-include-email)
+# $ export MYTAG=localbuild  ## you can choose whatever tag you like
+# $ docker tag torchaudio/smoketest 308535385114.dkr.ecr.us-east-1.amazonaws.com/torchaudio/smoke_test:${MYTAG}
+# $ docker push  308535385114.dkr.ecr.us-east-1.amazonaws.com/torchaudio/smoke_test:${MYTAG}
+
+FROM ubuntu:latest
+
+RUN apt-get -qq update && apt-get -qq -y install curl bzip2 \
```

Comment:
Consider putting this in a shell script?

Reply:
I'd prefer to use current format since there are just list of commands, no complex logic, still very readable to me.